api/v1/openapi.yaml

```diff
@@ -952,6 +952,21 @@ definitions:
         type: string
       log:
         type: string
+  Port:
+    description: Layer 4 port/protocol tuple
```

Comment:
Can we refer to "pair" instead? tuple is very networking heavy term and people draw conclusions. We don't want people to mistake this as both source destination tuple.